### ISSUE TITLE
[Feature Request] add more visibility to check description text

### ISSUE THREAD

**Issue by hesi2k** [amidaware/tacticalrmm#2013]
Is your feature request related to a problem? Please describe.

The text field of the description of the checks is cut off too short with “...”. Depending on the browser zoom, the unused distance to the next field increases even more (~50% cut at 100% zoom). 

Describe the solution you'd like

Perhaps it is possible to set the visible text to the automatic width so that the full column width is used.

![trmm_desc_zoom100](https://github.com/user-attachments/assets/08b914c5-2176-47d9-bbff-66d4856f76c6)

**Comment by silversword411** [comment]
I believe this was already improved

**Comment by wh1te909** [comment]
yup added in https://github.com/amidaware/tacticalrmm/issues/2014
